packages/key-management/src/util/ownSignatureKeyPaths.ts

```diff
@@ -108,7 +323,8 @@ const getRequiredSignersKeyPaths = (
 export const ownSignatureKeyPaths = async (
   txBody: Cardano.TxBody,
   knownAddresses: GroupedAddress[],
-  inputResolver: Cardano.InputResolver
+  inputResolver: Cardano.InputResolver,
+  dRepKeyHash: Crypto.Ed25519KeyHashHex
```

Comment:
Could just make this optional to not have to use `''` where this is not relevant

Reply:
Done here: [527cc7a](https://github.com/input-output-hk/cardano-js-sdk/pull/919/commits/527cc7a965a4094b538d270c276d0b7ad960365c)

Reply:
@mkazlauskas